OpenRA.Game/FieldLoader.cs

```diff
@@ -46,7 +46,7 @@ public MissingFieldsException(string[] missing, string header = null, string hea
 
 		public static Func<string, Type, string, object> InvalidValueAction = (s, t, f) =>
 		{
-			throw new InvalidOperationException("FieldLoader: Cannot parse `{0}` into `{1}.{2}` ".F(s, f, t));
+			throw new YamlException("FieldLoader: Cannot parse `{0}` into `{1}.{2}` ".F(s, f, t));
```

Comment:
It in five places: 
* OpenRA.Game/FileFormats/ReplayMetadata.cs
  * Added YamlException logging
* OpenRA.Game/GameInformation.cs
  * Added YamlException logging
* OpenRA.Game/Network/Connection.cs
  * No MiniYaml handling
* OpenRA.Game/Network/Session.cs
  * Added YamlException rethrow
* OpenRA.Mods.TS/UtilityCommands/LegacyTilesetImporter.cs
  * No MiniYaml loading